### DISCUSSION TITLE
Undo (Ctrl-Shift-T) close tab (Ctrl-W)

### DISCUSSION THREAD

**Discussion by myii** [TriliumNext/Trilium#1583] [Ideas]
I've had this a few times where I've hit `Ctrl-W` and closed a tab prematurely and then have to spend a while navigating to it again.  Having `Ctrl-Shift-T` would be exceptionally helpful.

**Comment by meichthys** [comment]
I know this is old, but this works now (at least on the desktop version) 👍
